recipes/biopet-bamstats/1.0/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Build file is copied from VarScan
+# https://github.com/bioconda/bioconda-recipes/blob/master/recipes/varscan/build.sh
+# This file was automatically generated by the sbt-bioconda plugin.
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
```

Comment:
version and build number are unnecessary. Conda will not install the same package in different versions into the same environment anyway.